lib/lrmd/proxy_common.c

```diff
@@ -263,7 +263,7 @@ remote_proxy_cb(lrmd_t *lrmd, const char *node_name, xmlNode *msg)
         crm_xml_add(request, PCMK_XE_ACL_ROLE, "pacemaker-remote");
 
         CRM_ASSERT(node_name);
-        pcmk__update_acl_user(request, PCMK__XA_LRMD_IPC_USER, node_name);
```

Comment:
We can probably drop the `PCMK__XA_LRMD_IPC_USER` constant... but currently it's still added via `remoted_proxy.c:ipc_proxy_dispatch()`

Reply:
Argh :(

I'm pretty sure that was never used. `crmd_remote_proxy_cb()` sets the ACL user to the remote node's name, which would override anything set here. (We don't use user-based ACLs from remote nodes, ACLs have to be defined for the node name instead.)